api/Controllers/HotelsController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using iTechArt.Hotels.Api.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace iTechArt.Hotels.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HotelsController : Controller
+    {
+        private readonly HotelsDatabaseContext _hotelsDb;
+
+        public HotelsController(HotelsDatabaseContext hotelsDb)
+        {
+            _hotelsDb = hotelsDb;
+        }
+
+        [HttpPost]
+        [Authorize(Roles = "admin")]
+        public async Task<IActionResult> CreateHotel([FromBody] Hotel request )
+        {
+            if (request == null)
+            {
+                return BadRequest("Wrong response");
```

Comment:
The message doesn't make sense for me. I already received `BadRequest` indicating that request was incorrect. Message should give me details to understand **why** it happened